registry/testcases/functional_testcases/test_service.py

```diff
@@ -614,134 +620,134 @@ def test_legal_approval_of_service(self):
-                        "url": "https://ropsten-marketplace-service-assets.s3.amazonaws.com/test_org_uuid/services/test"
-                               "_service_uuid/assets/20200212111248_proto_files.zip"
-                    }
-                },
-                "contributors": [{"email_id": "prashant@singularitynet.io"}],
-                "groups": [{"group_name": "default_group",
-                            "free_calls": 12,
-                            "free_call_signer_address": "0x7DF35C98f41F3Af0df1dc4c7F7D4C19a71Dd059F",
-                            "daemon_address": ["0x1234", "0x345"],
-                            "pricing": [
-                                {
-                                    "price_model": "fixed_price",
-                                    "price_in_cogs": 1,
-                                    "default": True
-                                }
-                            ],
-                            "endpoints": [
-                                "https://tz-services-1.snet.sh:8005"
-                            ],
-                            "test_endpoints": [
-                                "https://tz-services-1.snet.sh:8005"
-                            ],
-                            "group_id": "EoFmN3nvaXpf6ew8jJbIPVghE5NXfYupFF7PkRmVyGQ="
-
-                            }]
-            }
-            )
-        }
-        response = submit_service_for_approval(event=event, context=None)
-        assert (response["statusCode"] == 200)
-        response_body = json.loads(response["body"])
-        assert (response_body["status"] == "success")
-        assert (response_body["data"]["service_uuid"] == "test_service_uuid")
-        assert (response_body["data"]["service_state"]["state"] == ServiceStatus.APPROVAL_PENDING.value)
+    # @patch(
+    #     "registry.domain.services.service_publisher_domain_service.ServicePublisherDomainService.register_or_update_service_in_blockchain")
+    # @patch("common.utils.publish_zip_file_in_ipfs")
+    # @patch("common.ipfs_util.IPFSUtil.write_file_in_ipfs")
+    # @patch("common.utils.send_email_notification")
+    # @patch("common.utils.send_slack_notification")
+    # def test_submit_service_for_approval(self, slack_notification, email_notification, file_ipfs_hash, zip_file_ipfs_hash, blockchain_transaction):
+    #     blockchain_transaction.return_value = "0x2w3e4r5t6y7u8i9o0oi8u7y6t5r4e3w2"
+    #     zip_file_ipfs_hash.return_value = "Qwertyuiopasdfghjklzxcvbnm"
+    #     file_ipfs_hash.return_value = "Qzertyuiopasdfghjklzxcvbnn"
+    #     self.tearDown()
+    #     org_repo.add_item(
+    #         OrganizationDBModel(
+    #             name="test_org",
+    #             org_id="test_org_id",
+    #             uuid="test_org_uuid",
+    #             org_type="organization",
+    #             description="that is the dummy org for testcases",
+    #             short_description="that is the short description",
```

Comment:
why this is commented?

Reply:
It is not completed, currently working on submit_service_for_approval flow